sormas-ui/src/main/java/de/symeda/sormas/ui/utils/UserField.java

```diff
@@ -31,6 +32,8 @@ public class UserField extends CustomField<UserReferenceDto> {
 	private boolean readOnly;
 	private boolean enabled;
 	private List<UserReferenceDto> items = new ArrayList<>();
+	private boolean parentPseudonymizedFlag;
```

Comment:
this field is redundant, please remove it

Reply:
removed